.github/CONTRIBUTING.md

```diff
@@ -29,10 +32,28 @@ Also, have you signed the [Contribution License Agreement](https://cla.microsoft
 If you feel like your contribution is going to be a major effort, you should probably give us a heads-up. We have a lot of items captured in our backlog and we release every two weeks, so before you spend the time, just check with us to make
 sure your plans and ours are in sync :) Just open an issue on github and tag it "enhancement" or "feature request"
 
+### Commit messages
+
+This project follows the [Conventional Commits convention](https://www.conventionalcommits.org), meaning that your commits message should be structured as follows:
+
+```
+<type>[optional scope]: <description>
+[optional body]
+[optional footer]
+```
+
+The commit should contains the following structural elements:
```

Comment:
>The commit should contains the following structural elements [](start = 0, length = 60)

This makes it sound like the commit should contain all of these when in reality it the type should contain 1 of the following.